lib/index.js

```diff
@@ -365,6 +343,25 @@ function stringToBuffer(string) {
 }
 
 /**
+ *
+ * Converts an ArrayBuffer to a Buffer
+ *
+ * @api private
+ */
+
+function toBuffer(data) {
```

Comment:
This should be `arrayBufferToBuffer` or something like that, since we have `stringToBuffer` too.